### ISSUE TITLE
Skip generation of diagrams and leave only list of fields?

### ISSUE THREAD

**Issue by gumispg** [schemaspy/schemaspy#571]
Is there any way to skip generation of diagrams in schemaSpy 5.0.0? My goal is to just keep list of fields from objects because diagrams are not very useful for tables which have dozens of relationships but their generation takes a lot of time (even more than 10 hours in my case).

Thank you in advance.

**Comment by npetzall** [comment]
We don't support 5.0
We currently have no support for skipping relationships. 
Snapshot of 6.1.0 contains a feature to skip exported keys. Which was really time consuming on oracle databases but probably others as well. 

The closest thing to skipping relationships is to skip html and only get xml output.

**Comment by gumispg** [comment]
Can I propose an improvement on this? As you know for sure, it would be best to just have some optional flag to handle this if somebody just wants list of fields and no diagrams at all.

I'm quite happy to switch to higher version as soon as I can but the problem is lack of support for Salesforce databases (related database driver hasn't been updated for quite a long time). I know that drivers are a completely different story but the more databases supported by default we have, the more people will be engaged in development of this tool.

**Comment by npetzall** [comment]
Same driver as in #570?

**Comment by npetzall** [comment]
As for list of fields an no diagram can you give an example.

Xml output will give you that in a way. 

Would you still want the html for the database?

**Comment by gumispg** [comment]
@npetzall, yes, this is the same type of driver. I was just using slightly older version (2.2). More details can be found here: https://salesforce.stackexchange.com/questions/245590/schemaspy-6-0-0-generation-ends-up-with-null-pointer-exception

As for the list of fields without diagrams, yes, I thought about an html because we just upload output of the schemaSpy to our webserver so people from other teams can easily figure out what is the structure of our database. But our biggest reason to present output as static html pages is because some of our users are non-technical and they just need this information for reporting purposes and this is the only way for them to get it.

**Comment by npetzall** [comment]
Do you still want parent and child information in the fields list?

I have experimented with running graphviz in parallel with success. Using 4 threads instead of 1 should bring the time from 10hrs to 2.5hrs.

However the diagrams might still feel cluttered. 

So skipping diagrams will cut down on the time spent calling the dot executable. But there might be an expensive call getExportedKeys(). If using 6.1.0-SNAPSHOT you will get a message if the call will take a lot of time.

Don't have the ability at the moment to fix or look at this right know. But maybe in the coming week. Unless someone else jump at this.

**Comment by gumispg** [comment]
Sure, I don't expect this to be fixed right away, I just wanted to highlight this topic. Thank you very much for your help.

And yes, I wanted parent and child information in the fields list because Salesforce is quite specific here and these relationships are important to understand what is connected to what (especially if we would like to skip diagrams).
